### DISCUSSION TITLE
so-test shows no issues?

### DISCUSSION THREAD

**Discussion by te132064** [Security-Onion-Solutions/securityonion#4206] [Unsupported Versions] [answered]
I'm new to this so apologies if this is a stupid question or I missed something obvious.

I have a clean install of Security Onion and thought before I added it to my network and probably setup a port mirror from my main switch, I'd run so-test to see if anything came up and it all ran correctly but I did not then see anything appear in alerts or hunt. I have seen a few things appear in here but they were mainly innocuous so I though this would let all hell break lose but nothing.
It said it ran against bond0 which I am assuming is my monitoring interface but I didn't think this would matter as it was just an internal test. I was going to try some replays with so-import-pcap but thought I'd check this was as expected first?

**Comment by jertel** [comment]
Yes, `so-test` is expected to generate a lot of events, and alerts. After you run the command it will output a URL that you need to copy/paste into a browser. That URL is important because it includes the timeframe of the test events. Are you using that URL to see the test results?

**Reply by te132064** [reply]
I don't see any URL? I run the command at it looks just like the output from the video tutorial i.e. it shows some errors but does complete the replay. I then return to the alerts and it's identical. Is it maybe so-import-pcap you mean shows a URL?

I'll leave the system running for a while this time as it's possible I was too quick to say it didn't show and shut it down.

**Reply by te132064** [reply]
I tried an update and that made no difference but I then did a repeated install and the only thing I changed was the leave the network at the defaults which includes 172 and 10 rather than my home which is a 192 and then I see the replays. I guess they come from some other networks so it's all working now, thank you.

**Comment by te132064** [comment, marked as answer]
Reconfigure the network to the defaults as mine was set to 192.168.1.0/24 only.
